var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -207,10 +203,16 @@ def _def(variant, flag=None):
 
         if '+cuda' in spec:
             cuda_arch = spec.variants['cuda_arch'].value
-            if cuda_arch[0] != 'none':
-                args += [
-                    '-DCMAKE_CUDA_FLAGS=-arch=sm_{0}'.format(cuda_arch[0])
-                ]
+            if '@:6.999' in spec and not ('@develop' in spec):
```

Comment:
Can we consider this:
```suggestion
            if spec.satisfies('@:6.999'):
```
as an alternative, or are there reasons to have both checks like above? Asking since `develop` should behave like `+inf` and [be greater than any other numeric version](https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison).